panic: json: cannot unmarshal object into Go value of type dnfjson.packageSpecs when build anaconda-iso type disk image

1. attempted to build anaconda-iso disk image as below:
```
sudo podman run --rm -it --privileged --pull=newer --security-opt label=type:unconfined_t -v /var/lib/libvirt/images/output:/output quay.io/centos-bootc/bootc-image-builder:latest  --type anaconda-iso --tls-verify=false  quay.io/centos-bootc/centos-bootc:stream9 

```
2.  The build process was broken by bellow error message
```
[root@dell-xx-17 ~]# sudo podman run --rm -it --privileged --pull=newer --security-opt label=type:unconfined_t -v /var/lib/libvirt/images/output:/output quay.io/centos-bootc/bootc-image-builder:latest  --type anaconda-iso --tls-verify=false  quay.io/centos-bootc/centos-bootc:stream9 
Generating manifest manifest-anaconda-iso.json
panic: json: cannot unmarshal object into Go value of type dnfjson.packageSpecs

goroutine 1 [running]:
main.cmdBuild(0xc0001ce900?, {0xc0004b0500, 0x1, 0x4})
	/build/bib/cmd/bootc-image-builder/main.go:371 +0xcc5
github.com/spf13/cobra.(*Command).execute(0xc000004900, {0xc0004b04c0, 0x4, 0x4})
	/root/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:983 +0xabc
github.com/spf13/cobra.(*Command).ExecuteC(0xc000004600)
	/root/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1115 +0x3ff
github.com/spf13/cobra.(*Command).Execute(...)
	/root/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1039
main.run()
	/build/bib/cmd/bootc-image-builder/main.go:515 +0x786
main.main()
	/build/bib/cmd/bootc-image-builder/main.go:519 +0x13

```
![Screenshot from 2024-04-13 17-58-23](https://github.com/osbuild/bootc-image-builder/assets/17738600/90a60061-01a6-4cd6-86c5-a839fb9bc524)

--- a/Containerfile
+++ b/Containerfile
@@ -16,9 +16,7 @@ FROM registry.fedoraproject.org/fedora:39
 # - https://github.com/osbuild/osbuild/pull/1468
 COPY ./group_osbuild-osbuild-fedora-39.repo /etc/yum.repos.d/
 COPY ./package-requires.txt .
-RUN grep -vE '^#' package-requires.txt | xargs dnf install -y && rm -f package-requires.txt && \
-    dnf -y upgrade https://kojipkgs.fedoraproject.org//packages/rpm-ostree/2024.4/3.fc39/$(arch)/rpm-ostree-{,libs-}2024.4-3.fc39.$(arch).rpm \
-&& dnf clean all
+RUN grep -vE '^#' package-requires.txt | xargs dnf install -y && rm -f package-requires.txt && dnf clean all
 COPY --from=builder /build/bin/bootc-image-builder /usr/bin/bootc-image-builder
 COPY entrypoint.sh /
 

--- a/package-requires.txt
+++ b/package-requires.txt
@@ -2,7 +2,7 @@
 # from the Containerfile by default, using leading '#' as comments.
 
 # This project uses osbuild
-osbuild osbuild-ostree osbuild-depsolve-dnf
+osbuild-115-1.20240410231807828982.main.2.gdbe70396.fc39 osbuild-ostree-115-1.20240410231807828982.main.2.gdbe70396.fc39 osbuild-depsolve-dnf-115-1.20240410231807828982.main.2.gdbe70396.fc39
 
 # We mount container images internally
 podman

